python_modules/dagster/dagster/_core/storage/captured_log_manager.py

```diff
@@ -258,3 +272,123 @@ def unsubscribe(self, subscription: CapturedLogSubscription) -> None:
     def build_log_key_for_run(self, run_id: str, step_key: str) -> Sequence[str]:
         """Legacy adapter to translate run_id/key to captured log manager-based log_key."""
         return [run_id, "compute_logs", step_key]
+
+    def get_log_keys_for_log_key_prefix(
+        self, log_key_prefix: Sequence[str]
+    ) -> Sequence[Sequence[str]]:
+        """Returns the logs keys for a given log key prefix. This is determined by looking at the
+        directory defined by the log key prefix and creating a log_key for each file in the directory.
+        """
+        # NOTE: This method was introduced to support backfill logs, which are always stored as .err files.
+        # Thus the implementations only look for .err files when determining the log_keys. If other file extensions
```

Comment:
is there any reason why the initial implementation shouldn't also look for `.out` files?

Reply:
not really. mostly just reducing surface area to test against, but it wouldn't be hard to add. I have some follow up PRs planned for this stack, so i'll add expanding to .out files as part of that so it doesn't block the backfill work from landing